### ISSUE TITLE
Accessibility issues with CAP table

### ISSUE THREAD

**Issue by mitchelbaker-cisa** [cisagov/ScubaGear#1047]
# 🐛 Summary #

There's a couple areas where the CAP table could be improved for accessibility purposes.

1. For each row there are chevron arrows with alt/title attributes both set to "Show more" or "Show less". Currently the alt/title attributes are read aloud with a screen reader as "Show more show more" which is redundant. It would help to include the CAP name here for additional info, ie "Show more for the Live - Session Length SHALL be Limited policy". This would also make use of a screen reader easier. 
2. The chevron arrows are image tags, so they can't be focused on with the tab key. [For WCAG compliance all functionality should be operable with a keyboard interface.](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html)  This also applies to "..." throughout the CAP table which when clicked expands the clicked row. 

## To reproduce ##

1. [Download NVDA](https://www.nvaccess.org/download/) for Windows or some other screen reader depending on your OS. Enable it. 
2. Generate a report for AAD 
3. Press tab to reach the CAP section of the report
4. Hover mouse over chevron icons to hear the repeated alt/title attributes

## Expected behavior ##

Users should be able to focus chevron arrows in the CAP table, and expand/collapse each row individually. This should also apply to "..." displayed throughout which also acts as a way to expand the given row. 

## Any helpful log output or screenshots ##

![Screenshot (68)_LI](https://github.com/cisagov/ScubaGear/assets/149098823/769d663c-b6e3-4a83-bce3-2ee8cbfd72f7)
